spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh

```diff
@@ -63,8 +64,17 @@ describe_vcr () {
   echo "== oc get all --show-all --all-namespaces -o wide --show-labels =="
   oc get all --show-all --all-namespaces -o wide --show-labels
   echo
-  echo "== oc get images =="
-  oc get images
+  echo "== oc get resourceQuotas --all-namespaces --show-kind --show-labels =="
+  oc get resourceQuotas --all-namespaces --show-labels
+  echo
+  echo "== oc get persistentvolumes --show-kind --show-labels =="
+  oc get persistentVolumes --show-labels
+  echo
+  echo "== oc get persistentvolumeclaims --all-namespaces --show-kind --show-labels =="
+  oc get persistentVolumeClaims --all-namespaces --show-labels
```

Comment:
We have a diff of `--show-kind` in the three echo. seems helpful to include, event if we are only showing one object kind at a time.